spec/services/users/delete_articles_spec.rb

```diff
@@ -14,6 +14,13 @@
     expect(Article.find(article3.id)).to be_present
   end
 
+  it "deletes the articles' discussion locks before deleting the article" do
+    create(:discussion_lock, article: article, locking_user: user)
+    expect do
+      described_class.call(user)
+    end.to change(DiscussionLock, :count).from(1).to(0)
```

Comment:
@Zhao-Andy This is fine, but I often prefer to use a single `by(-1)` assertion as that's less likely to break due to unrelated reasons.